test/fork/basic-functionality-tests.js

```diff
@@ -1,331 +1,560 @@
+      ),
+      this.abMembers,
+      this.governance,
+    );
+  });
 
-    const balancePreSell = await web3.eth.getBalance(Address.NXMHOLDER);
-    const sellTx = await pool.sellNXM(nxmOut, '0', { from: Address.NXMHOLDER });
+  it.skip('vote for the ybDAI claim', async function () {
+    const [manager] = this.abMembers;
+    const assessmentStakingAmount = parseEther('1000');
 
-    const { gasPrice } = await web3.eth.getTransaction(sellTx.receipt.transactionHash);
-    const ethSpentOnGas = Decimal(sellTx.receipt.gasUsed).mul(Decimal(gasPrice));
-    const balancePostSell = await web3.eth.getBalance(Address.NXMHOLDER);
-    const ethOut = toDecimal(balancePostSell).sub(toDecimal(balancePreSell)).add(ethSpentOnGas);
-    const ethInDecimal = toDecimal(maxBuy);
+    await this.assessment.connect(manager).stake(assessmentStakingAmount);
+    await this.assessment.connect(manager).castVotes(ybDaiAssessmentId, [true], ['Assessment data hash'], 0);
+  });
 
-    assert(ethOut.lt(ethInDecimal), 'ethOut > ethIn');
+  it.skip('vote for the ybETH claim', async function () {
+    const [manager] = this.abMembers;
+    const assessmentStakingAmount = parseEther('1000');
 
-    console.log({
-      ethOut: toDecimal(ethOut).div(1e18).toString(),
-      ethIn: ethInDecimal.div(1e18).toString(),
-    });
+    await this.assessment.connect(manager).stake(assessmentStakingAmount);
+    await this.assessment.connect(manager).castVotes(ybEthAssessmentId, [true], ['Assessment data hash'], 0);
   });
 
   it('sets DMCI to greater to 1% to allow floor increase', async function () {
-    const { voters, governance, mcr, whales } = this;
+    const newMaxMCRFloorChange = BigNumber.from(100);
 
-    const newMaxMCRFloorIncrement = toBN(100);
-    const parameters = [
-      ['bytes8', hex('DMCI')],
-      ['uint', newMaxMCRFloorIncrement],
-    ];
-
-    const updateParams = web3.eth.abi.encodeParameters(
-      parameters.map(p => p[0]),
-      parameters.map(p => p[1]),
-    );
+    const DMCI = toBytes('DMCI', 8);
 
     await submitMemberVoteGovernanceProposal(
-      ProposalCategory.upgradeMCRParameters,
-      updateParams,
-      [...voters, ...whales],
-      governance,
+      PROPOSAL_CATEGORIES.upgradeMCRParameters,
+      defaultAbiCoder.encode(['bytes8', 'uint'], [DMCI, newMaxMCRFloorChange]),
+      [...this.abMembers, ...this.members], // add other members
+      this.governance,
     );
 
-    const maxMCRFloorIncrement = await mcr.maxMCRFloorIncrement();
+    const maxMCRFloorAfter = await this.mcr.maxMCRFloorIncrement();
 
-    assert.equal(maxMCRFloorIncrement.toString(), newMaxMCRFloorIncrement.toString());
+    expect(maxMCRFloorAfter).to.be.equal(newMaxMCRFloorChange);
   });
 
-  it('triggers MCR update after ETH injection to the pool to MCR% > 130%', async function () {
-    const { mcr, pool } = this;
+  it('performs hypothetical future Governance upgrade', async function () {
```

Comment:
let's add a hypothetical pool upgrade with token asserts

you can copy paste that from the bit branch that's in progress in terms of token balances